### PR TITLE
feat: add operation id to http response

### PR DESCRIPTION
**Pull request by stijnmoreels**
Adds operation ID to the HTTP response message.

### CODE REVIEW

File: docs/preview/features/correlation.md [hunk 1]
```diff
@@ -7,21 +7,6 @@ layout: default
...
-    - [Sending side](#sending-side)
-    - [Receiving side](#receiving-side)
-  - [Configuration](#configuration)
-    - [Configuring HTTP correlation services](#configuring-http-correlation-services)
-    - [Configuring HTTP correlation client tracking](#configuring-http-correlation-client-tracking)
-  - [Dependency injection](#dependency-injection)
-  - [Logging](#logging)
-  - [Using correlation within Azure Functions](#using-correlation-within-azure-functions)
-    - [Installation](#installation-1)
-    - [Usage](#usage-1)
```
Comment (pim-simons) [inline docs/preview/features/correlation.md:23]: why was this part removed?

**Review by pim-simons — APPROVED** [review]
looks good to me

File: docs/preview/features/correlation.md (hunk 1 above)
Comment (stijnmoreels) [reply docs/preview/features/correlation.md:23]: Ah, since we already have this automatically generated in the right-side sidebar. All the titles and subtitles are already there so we don't have to duplicate it in the page itself.
LIke:
image
